lib/release_notes_bot/persists.ex

```diff
@@ -193,6 +194,15 @@ defmodule ReleaseNotesBot.Persists do
     end
   end
 
+  defp update_confluence_page(data) do
+    headers = get_headers(data.token)
+    {:error, "Not implemented"}
+    # curl -u admin:admin -X PUT -H 'Content-Type: application/json' -d '{"id":"3604482","type":"page",
+    # "title":"new page","space":{"key":"TST"},"body":{"storage":{"value":
+    # "<p>This is the updated text for the new page</p>","representation":"storage"}},
+    # "version":{"number":2}}' http://localhost:8080/confluence/rest/api/content/3604482
+  end
+
```

Comment:
Let's be conscious as to not randomly move code in unrelated commits. If this is truly the location you desire for this function, then introduce it here initially.